### ISSUE TITLE
Feature request: annotations/events marks

### ISSUE THREAD

**Issue by igorvlassov** [PhilJay/MPAndroidChart#2676]
Dear Sir,
Could you implement annotations/events please? IMHO they are very common charts feature nowadays and should be provided out of the box. There is example:
https://www.amcharts.com/demos/stock-events/

**Comment by StuStirling** [comment]
Agreed that this is a very important feature. I need this to be able to use the library

**Comment by tommimarkus** [comment]
you can probably fake these by using the scatterdataset with the linedataset in combineddata
